core/Console/DialogHelper.php

```diff
@@ -0,0 +1,505 @@
+<?php
+
+// This file is a copy of DialogHelper from symfony console 2.8
```

Comment:
You may want to use the "new"  [QuestionHelper](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Helper/QuestionHelper.php) instead

Reply:
I have readded that one for BC only. Some community plugins might still use the DialogHelper, and in order not to break them, we would have this class, at least for a while. But the plan would be to convert our usages. Guess that would be something we could already do before this is merged as iirc the QuestionHelper was also available in 2.6

Reply:
It make sense, but I'm not sure we should convert these commands in that contribution because it will make it too big and maybe difficult to be accepted/reviewed.

Once this one is merged, I'll give a hand to help : I don't know the Matomo code base but this will be a perfect topic for beginner contributions, right ? 😇

Reply:
I guess we could actually already start to convert the commands. At least replacing DialogHelper with QuestionHelper should be doable without that PR merged, right?

Reply:
Yes 👍